types/reactstrap/lib/Dropdown.d.ts

```diff
@@ -10,9 +10,6 @@ export type Direction =
 export interface UncontrolledDropdownProps extends React.HTMLAttributes<HTMLElement> {
     [key: string]: any;
     isOpen?: boolean;
-
-    toggle?: React.KeyboardEventHandler<any> | React.MouseEventHandler<any>;
```

Comment:
This toggle does exist.

https://github.com/reactstrap/reactstrap/blob/de5f4822ce7d1505a02c775b7396bb15f0970801/src/Dropdown.js#L21

It does have events originating from a [keypress](https://github.com/reactstrap/reactstrap/blob/de5f4822ce7d1505a02c775b7396bb15f0970801/src/Dropdown.js#L131) and a [click](https://github.com/reactstrap/reactstrap/blob/de5f4822ce7d1505a02c775b7396bb15f0970801/src/Dropdown.js#L111)